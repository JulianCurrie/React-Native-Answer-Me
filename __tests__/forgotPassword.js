import 'react-native';
import React from 'react';
import ForgotPassword from '../src/components/auth/forgotPassword';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <ForgotPassword />
  );
});
