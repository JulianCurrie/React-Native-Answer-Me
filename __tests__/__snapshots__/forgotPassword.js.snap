exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "azure",
      "flex": 1,
      "justifyContent": "center",
      "padding": 40,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "red",
        "textAlign": "center",
      }
    } />
  <TextInput
    onChangeText={[Function]}
    placeholder="Email"
    style={
      Object {
        "alignSelf": "center",
        "borderColor": "black",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 50,
        "marginTop": 10,
        "textAlign": "center",
        "width": 250,
      }
    } />
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "column",
        "justifyContent": "space-between",
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "blue",
            "marginTop": 10,
          }
        }>
        Back
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Send Reset Email
      </Text>
    </View>
  </View>
</View>
`;
